plugins/hns.yaml

```diff
@@ -18,12 +18,12 @@ spec:
     HNC is controlled via regular Kubernetes objects, but this plugin makes it
     easy to create subnamespaces, arrange regular (full) namespaces into
     hierarchies, and configure HNC to propagate different kinds of objects.
-  version: v0.7.0
+  version: v0.8.0
   caveats: |
     This plugin requires the most recent minor version of HNC on your cluster. Get it at:
-      https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.7.0
+      https://github.com/kubernetes-sigs/multi-tenancy/releases/tag/hnc-v0.8.0
```

Comment:
If you find a way to use a stable link here, such version bumps could be auto-approved.